pkg/fakerp/fakerp.go

```diff
@@ -60,9 +62,17 @@ func createOrUpdate(ctx context.Context, log *logrus.Entry, cs, oldCs *api.OpenS
 		return nil, kerrors.NewAggregate(errs)
 	}
 
-	template, err := GetPluginTemplate()
-	if err != nil {
-		return nil, err
+	var err error
+	var template *pluginapi.Config
+	if shared.IsUpdate() && shared.IsScaleOperation(cs, oldCs) {
```

Comment:
This might do for now. But there are no limitations that we could not do scale-up for a customer using adminAPI. It is just logic ordering. But we should track which API endpoint request hit and act accordingly.

Reply:
+ `IsUpdate` is little bit useless if we start thinking about scaleUp only and updates. ll this is fakeRP code, but logic looks weak for all this :/ dont know how to fix. need to think ...